pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -525,7 +532,7 @@ func performClusterHealthCheck(c client.Client, metricsClient metrics.Metrics, c
 	if len(ic) > 0 {
 		icQuery = `,alertname!="` + strings.Join(ic, `",alertname!="`) + `"`
 	}
-	healthCheckQuery := `ALERTS{alertstate="firing",severity="critical",namespace=~"^openshift.*|^kube.*|^default$",namespace!="openshift-customer-monitoring"` + icQuery + "}"
+	healthCheckQuery := `ALERTS{alertstate="firing",severity="critical",namespace=~"^openshift.*|^kube.*|^default$",namespace!="openshift-customer-monitoring|openshift-logging"` + icQuery + "}"
```

Comment:
Still having problems with this query. It needs to use the regex exquivalent of != as in the suggestion below:
```suggestion
	healthCheckQuery := `ALERTS{alertstate="firing",severity="critical",namespace=~"^openshift.*|^kube.*|^default$",namespace!~"openshift-customer-monitoring|openshift-logging"` + icQuery + "}"
```